src/favorites/FavoriteChips.tsx

```diff
@@ -39,52 +39,51 @@ export const FavoriteChips: React.FC<Props> = ({
   onAddFavorite,
 }) => {
   const {favorites} = useFavorites();
-  const styles = useStyles();
   const {t} = useTranslation();
   const {onCurrentLocation} = useCurrentLocationChip(onSelectLocation);
   const disableMap = useDisableMapCheck();
+  const styles = useStyles();
   const activeType = (type: ChipTypeGroup) => chipTypes.includes(type);
 
   return (
     <>
-      {(activeType('location') || activeType('map')) && (
-        <View style={[style, styles.staticChipsContainer]}>
-          {activeType('location') && (
-            <FavoriteChip
-              interactiveColor={themeColor}
-              mode="secondary"
-              text={t(FavoriteTexts.chips.currentLocation)}
-              accessibilityRole="button"
-              accessibilityHint={chipActionHint ?? ''}
-              leftIcon={{svg: LocationIcon}}
-              onPress={onCurrentLocation}
-              testID="currentLocationChip"
-            />
-          )}
-          {activeType('map') && !disableMap && (
-            <FavoriteChip
-              text={t(FavoriteTexts.chips.mapLocation)}
-              accessibilityRole="button"
-              leftIcon={{svg: Pin}}
-              onPress={onMapSelection}
-              interactiveColor={themeColor}
-              mode="secondary"
-              testID="mapLocationChip"
-            />
-          )}
-        </View>
-      )}
       <ScrollView
         horizontal={true}
         showsHorizontalScrollIndicator={false}
         keyboardShouldPersistTaps="handled"
-        contentContainerStyle={
+        contentContainerStyle={[
           (activeType('favorites') && favorites.length > 0) ||
           activeType('add-favorite')
             ? style
-            : undefined
-        }
+            : undefined,
+          style,
```

Comment:
If adding style anyway, there is no need for the conditional check above. Or did you mean to remove line 59?